aws/sdk/integration-tests/s3/tests/request_information_headers.rs

```diff
@@ -168,7 +170,7 @@ async fn three_retries_and_then_success() {
 //     let config = aws_sdk_s3::Config::builder()
 //         .credentials_provider(Credentials::for_tests())
 //         .region(Region::new("us-east-1"))
-//         .http_connector(DynConnector::new(conn.clone()))
+//         .http_client(DynConnector::new(conn.clone()))
```

Comment:
Is this correct to pass a `DynConnector` to `.http_client` ?

Reply:
It isn't, but I'm not going to spend time fixing these commented out tests.

Reply:
Looks like we didn't have a tracking issue to fix these, so I filed one: https://github.com/awslabs/smithy-rs/issues/3030